### ISSUE TITLE
No "Extruded Filament" shown ("0" value shown during print)

### ISSUE THREAD

**Issue by energiebrand** [StefanCohen/OctoPrint-Dashboard#164]
Same configuration as issue #163 but now I noticed that also the "Extruded Filament" stucks at "0".
![Extruded filament](https://user-images.githubusercontent.com/60507600/82821591-ac413080-9ea4-11ea-97d1-a315a257eadd.jpg)

**Comment by StefanCohen** [comment]
Thanks for the feedback. Do you get any javascript errors when this happens?

**Comment by StefanCohen** [comment]
I think Ollie found a possible explanation for this in #160 . His problem was that psutil crashed on his system causing the backend to stop sending data to the frontend. This is likely what causes #163 as well. I don't know why this happens on your RPi since I can't reproduce it here but I can try to add some error handling around the call to psutil to mitigate or at least detect and log the error.

**Comment by energiebrand** [comment]
...no javascript errors... are shown.
Today checked on a RPI 3B+ with Octoprint 1.4.0 on OctoPi 0.17.0 with same behaviour as on RPi 3B  with Octoprint 1.4.0 on OctoPi 0.16.0

**Comment by energiebrand** [comment]
This is my comment from the issue I opened in Display Layer Progress Plugin:

Octoprint 1.4.0 on Octopi 0.17.0 and Octoprint 1.4.0 on Octopi 0.16.0
I can confirm that issue #148 (Display Layer Progress) can be closed from my point of view.
Also from my point of view **issue #163 #164 of the Dasboard can be closed. I propose to close issue #160 in Dashboard.**

Octoprint 1.4.0 on Octopi 0.17.0
What did I?
Removed Dashboard, Display Layer Progress and Custom Control Editor including data.
Sudo apt-get update & upgrade (just routine...)
Installed Dashboard, Display Layer Progress and Custom Control Editor (all latest revisions)
Defined Custom control buttons in Custom Control Editor, saved, shut off, boot
Now everything works, all problems are gone!

Octoprint 1.4.0 on Octopi 0.16.0
Did the same as above but -> the issue #31 & 32 of the Custom Control Editor are still present
I did a fallback to the old release of Custom Control Editor; here it works!
However no more problems in combination with Dashboard and Display Layer Progress!
